static/build.html

```diff
@@ -2,32 +2,34 @@
 <html lang="en" prefix="og: http://ogp.me/ns#">
     <head>
         <meta charset="utf-8"/>
-        <title>Build | GrapheneOS</title>
-        <meta name="description" content="Building instructions for GrapheneOS, a security and privacy focused mobile OS with Android app compatibility."/>
+        <title>GrapheneOS</title>
```

Comment:
It seems that you're unintentionally clobbering each of the metadata sections with the one for the index page.